src/objects/core/zcl_abapgit_objects_check.clas.abap

```diff
@@ -158,6 +177,33 @@ CLASS zcl_abapgit_objects_check IMPLEMENTATION.
   ENDMETHOD.
 
 
+  METHOD warning_overwrite_find_all.
+
+    DATA: ls_overwrite LIKE LINE OF rt_overwrite.
+
+    FIELD-SYMBOLS: <ls_result> LIKE LINE OF it_results.
+
+    LOOP AT it_results ASSIGNING <ls_result> WHERE NOT obj_type IS INITIAL
+      AND NOT ( lstate IS INITIAL AND rstate IS INITIAL ).
```

Comment:
I have trouble understanding this condition in relation to the IF statement below, suggest sacrificing the performance, which will make it easier to understand, and maintain as there will only be one condition instead of two that must match

```suggestion
    LOOP AT it_results ASSIGNING <ls_result> WHERE NOT obj_type IS INITIAL.
```

Reply:
alternatively move everything to the loop if possible

Reply:
i will combine the warning methods and add field `local_change (abap_bool)`. if `local_change` is false, then the entry will not be relevant for a regular pull (because nothing was changed locally).

Reply:
cool, I think that will look better, instead of the boolean input parameter

the boolean will still be part of the gui_deserialize I guess, but its okay IMHO